test/end-to-end/specs/viewBlueprint.test.js

```diff
@@ -220,6 +220,8 @@ describe("View Blueprint Page", function() {
       });
 
       it('should show "Complete"', function() {
+        // set test timeout to 30 minutes for image building case ONLY
+        this.timeout(1800000);
         expect(viewBlueprintPage.completeLebel.getText()).to.equal("Complete");
```

Comment:
This now builds an actual image, but that doesn't get verified at all. As an initial step, I suggest to ensure that the file exists at the expected place, and that running `qemu info` on it gives the expected virtual size.

Reply:
To clarify: As a further step, the test should actually boot the generated image, with qemu. For that it needs to install and enable some network-accessible service ssh, so that we can validate from outside that the image boots and works, and even login and inspect it. But this is fine for a follow-up PR.

Reply:
I'm still convinced that there should be *some* validation of the created image, to cover a complete end-to-end workflow. But this can happen in a follow-up.